src/ResourceManager/KeyVault/Commands.KeyVault/help/Add-AzureKeyVaultKey.md

```diff
@@ -12,176 +12,36 @@ Creates a key in a key vault or imports a key into a key vault.
 
 ## SYNTAX
 
-### Create (Default)
```

Comment:
@cormacpayne help generation seems to output incorrect content for a few commands, I haven't figured out why. I've manually updated some of them, and this one slipped through; will revert the change.